### ISSUE TITLE
FID指标

### ISSUE THREAD

**Issue by NuanBaobao** [cure-lab/MagicDrive#32]
Thank you very much for your work, but I would like to ask a question about FID. I used the weights you published to generate a validation set data of 224 * 400, and processed the real validation set the same way, saving it as a PNG format image. However, after calculating the FID, it was much higher than the data in the paper. How did you calculate it? Thank you.

**Comment by flymin** [comment]
We use [pytorch-fid](https://github.com/mseitzer/pytorch-fid).

**Comment by NuanBaobao** [comment]
Thank you for your response and help! I will follow your method and use pytorch-fid to calculate the FID. If I have any other questions, I will reach out to you again. Thanks once more!

**Comment by ppeterpp** [comment]
i used pytorch-fid and the generated pics from the released weights also yielded much higher FID

**Comment by NuanBaobao** [comment]
I am as well. Using pytorch-fid, I measured metrics that can reach around 25.

**Comment by ppeterpp** [comment]
@flymin is it that your released model wasn't the one leading to the reported FID, or anything wrong with the reproduction process described in the issue? Please shed some light!

**Comment by flymin** [comment]
There is no problem with the description; the released model&weight is the one we used for testing in the paper. I cannot suspect any reason at the moment. I will try to release more resources soon.

**Comment by zhaoyangwei123** [comment]
> @flymin is it that your released model wasn't the one leading to the reported FID, or anything wrong with the reproduction process described in the issue? Please shed some light!

May I ask how you generate a new test set to calculate FID?

**Comment by flymin** [comment]
Just want to let all you guys know that I am working on this. Please watch progress in this issue.

**Comment by flymin** [comment]
Hi all, code is ready now. Enjoy!
